src/NuGet.Core/NuGet.Frameworks/NuGetFrameworkFactory.cs

```diff
@@ -168,26 +146,15 @@ private static string[] GetParts(string targetPlatformMoniker)
         /// </summary>
         public static NuGetFramework ParseFrameworkName(string frameworkName, IFrameworkNameProvider mappings)
         {
-            if (frameworkName == null)
-            {
-                throw new ArgumentNullException(nameof(frameworkName));
-            }
-
-            if (mappings == null)
-            {
-                throw new ArgumentNullException(nameof(mappings));
-            }
-
-            var parts = frameworkName.Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries).Select(s => s.Trim()).ToArray();
+            if (frameworkName == null) throw new ArgumentNullException(nameof(frameworkName));
+            if (mappings == null) throw new ArgumentNullException(nameof(mappings));
 
-            NuGetFramework result = null;
+            string[] parts = frameworkName.Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries).Select(s => s.Trim()).ToArray();
```

Comment:
Do we have a `static readonly char[]` with a single comma somewhere we could re-use?  I'd imagine this is a hot path and seeing us allocate a char array for every call makes me sad.

Reply:
There are 20 examples in our code of `new char[] { ',' }` 😱.  I'll fix this one here, but I don't want this PR to touch files outside of NuGet.Frameworks.